src/CMakeLists.txt

```
@@ -69,6 +69,18 @@ else()
   )
 endif()
 
+# unset these two if they are cached
+unset(MINIZIP_FOUND)
+unset(MINIZIP_STATIC_LIBRARIES)
```

This still links to the system minizip after enabling OPTION_FORCE_EMBEDDED_MINIZIP in an existing build dir.
This worked for me:
```suggestion
unset(MINIZIP_FOUND)
set(MINIZIP_STATIC_LIBRARIES "")
```